api/src/app/modules/submission/submission.service.ts

```diff
@@ -668,13 +668,39 @@ export class SubmissionService extends DataService<Submission, Repository<Submis
     }
   }
 
+  private async simplifyGeometry(submissionID: number, spatialObjectCode: SpatialObjectCodeEnum) {
+    this.logger.debug(`Simplify geometry for submittion: ${submissionID}, spatial object type: ${spatialObjectCode}`);
+    try {
+      await this.getDataSource()
+        .query(
+          `
+            UPDATE app_fom.${SpatialObjectTableEnum[spatialObjectCode]} SET geometry=ST_SimplifyPreserveTopology(geometry, 2.5) where submission_id = ${submissionID};
```

Comment:
There is another way to do this to avoid doing an update. See the logic that does the transformation between coordinate systems - a similar method could be used to do the simplify.

I'd add a comment for the 2.5 constant that this was established by Forestry Geospatial experts as the standard accuracy level for forestry applications.